client/settings/notices/legacy-experience-transition/index.js

```diff
@@ -0,0 +1,111 @@
+import { useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import styled from '@emotion/styled';
+import React from 'react';
+import { Button } from '@wordpress/components';
+import { recordEvent } from 'wcstripe/tracking';
+
+const NoticeWrapper = styled.div`
+	display: flex;
+	flex-direction: column;
+	padding: 12px 16px;
+	align-items: flex-start;
+	gap: 12px;
+	margin: 0 0 24px 0;
+	background: #fcf9e8;
+`;
+
+const Message = styled.div`
+	color: #1e1e1e;
+	font-size: 13px;
+`;
+
+const DisableLegacyButton = styled( Button )`
+	box-shadow: inset 0 0 0 1px #bd8600 !important;
+	color: #bd8600 !important;
+`;
+
+const LearnMoreAnchor = styled.a`
+	color: #bd8600 !important;
+`;
+
+const LegacyExperienceTransitionNotice = ( {
+	isUpeEnabled,
+	setIsUpeEnabled,
+} ) => {
```

Comment:
Thanks for the review Mayisha!

That sounds good. I decided to have these properties as parameters for the LegacyExperienceTransitionNotice component because we're already using the UpeToggleContext in the parent component (link below). But glad to handle it differently!

https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3014/files/78e0ee959197071eab59fd71747c10e4394a5821#diff-9ea1e829339122266bbb6d0807f9d81ba8128ade1c14da9648af5aac07c58dbbR42